fdbrpc/AsyncFileKAIO.actor.h

```diff
@@ -794,10 +794,15 @@ ACTOR Future<Void> runTestOps(Reference<IAsyncFile> f, int numIterations, int fi
 }
 
 TEST_CASE("/fdbrpc/AsyncFileKAIO/RequestList") {
-	if(!g_network->isSimulated()) { // This test does nothing in simulation because simulation doesn't support AsyncFileKAIO
+	if (!g_network
+	         ->isSimulated()) { // This test does nothing in simulation because simulation doesn't support AsyncFileKAIO
```

Comment:
Can you fix the horrible wrapping done by clang-format?

Reply:
Done